src/js/hooks/productSupplier/form/useProductSupplierForm.js

```diff
@@ -126,14 +127,29 @@ const useProductSupplierForm = () => {
 
   // eachPrice is a computed value from packagePrice and productPackageQuantity
   useEffect(() => {
-    if (packagePrice && productPackageQuantity) {
+    if (
+      !_.isNil(packagePrice) &&
+      !_.isNil(productPackageQuantity) &&
+      productPackageQuantity !== 0
+    ) {
       setValue('eachPrice', decimalParser(packagePrice / productPackageQuantity, 4));
     } else {
       setValue('eachPrice', '');
     }
   },
   [packagePrice, productPackageQuantity]);
 
+  const uom = useWatch({ control, name: 'uom' });
+
+  // preselect value 1 when unit of measure Each is selected
+  useEffect(() => {
+    if (uom?.id === 'EA') {
```

Comment:
this feels dangerous to me. Are we sure that on every instance this would have the id of `'EA'`? 
I think we'd be safer if we looked at `uom.code`, but we would have to include the code in the response of uoms and change the `getDefaultValues()` to include the code.
Unless @awalkowiak / @jmiranda know if this will always have id of `'EA'` on any instance, then we can let it go.